src/sagemaker/fw_utils.py

```diff
@@ -148,7 +148,15 @@
 ]
 
 
-TORCH_DISTRIBUTED_SUPPORTED_FRAMEWORK_VERSIONS = ["1.11", "1.11.0"]
+TORCH_DISTRIBUTED_TRAINIUM_SUPPORTED_FRAMEWORK_VERSIONS = [
```

Comment:
The corresponding unit test files need to be updated like https://github.com/aws/sagemaker-python-sdk/blob/ed69db250ce74eff7476a7a20085ba47f092134d/tests/unit/test_fw_utils.py#L951-L1042.

Reply:
thanks for the comment, added test cases